content/tech-ops/team_device_usage_privacy.md

```diff
@@ -0,0 +1,25 @@
+# Team device usage privacy
+
+Sourcegraph is a high-trust and [high-agency](../company/values.md#high-agency) company. We must trust one another to be operating in the best interests of the team.
+
+Privacy is a right that we believe deeply in, both for our customers (e.g. see our [philosophy on data collection from self-hosted Sourcegraph instances](https://docs.sourcegraph.com/dev/background-information/adding_ping_data#ping-philosophy)) and for our teammates. **Nobody, from Tech Ops to your manager to the CEO, cares how you use your Sourcegraph device or when you work, as long as you are using your device to perform your job duties, hitting your goals, and keeping company and customer data secure.**
```

Comment:
_"Nobody cares how you use your Sourcegraph device"_

Is this a legally sound statement? Does it not matter to Sourcegraph if I do illegal activities on the company device?
For instance, if we had a client in the film/tv industry and teammates torrented their content it would not be a good situation.

All probably obvious and logical, but might be worth rephrasing this sentence a bit, so that the "your job duties" part is more explicit. Right now it doesn't read like an exclusive use expectation.